tutorials/java/chapters/ch07.adoc

```diff
@@ -0,0 +1,221 @@
+:imagesdir: images
```

Comment:
Chapter 7 needs to be reworked and not included in the TOC. Do you recommend to remove it?